broker-daemon/state-machines/order-state-machine.js

```diff
@@ -280,7 +289,8 @@ const OrderStateMachine = StateMachine.factory({
       }
 
       const errHandler = (e) => {
-        this.reject(e)
+        const relayerError = new Error(ORDER_ERROR_CODES.RELAYER_UNAVAILABLE)
```

Comment:
Is this the only error that's possible? What's in the `e` parameter here?

Reply:
Good question. When the gRPC stream ends from with the relayer it sends an error code 14, unavailable. Added a conditional to only add the `relayerError` when the broker receives code 14.